.env.example

```diff
@@ -22,3 +22,4 @@ SMTP_FROM=
 NEXTAUTH_JWT_SIGNING_PRIVATE_KEY=
 NEXTAUTH_URL=
 NEXTAUTH_SECRET=
+NEXTAUTH_ACL="boxyhq.com"
```

Comment:
This should be an array of emails or glob patterns. Like `*@boxyhq.com,deepak@somedomain.com`